testsuite/cluster-test/src/instance.rs

```diff
@@ -80,6 +80,42 @@ struct K8sInstanceInfo {
     kube: ClusterSwarmKube,
 }
 
+impl InstanceConfig {
+    pub fn replace_tag(&mut self, new_tag: String) -> Result<()> {
+        match self {
```

Comment:
since self is already &mut, I don't think you need `ref mut` in the match handles

Reply:
Yep, you are right